Firmware/.vscode/settings.json

```diff
@@ -3,6 +3,8 @@
     "files.exclude": {
         "build": true
     },
+    "C_Cpp.intelliSenseEngineFallback": "Enabled",
+    "C_Cpp.intelliSenseEngine": "Tag Parser",
```

Comment:
I'd like to get the Default engine working, the tag parser just gives me every single name ever, with no intelligence.
Can you see if you can make it work?
https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/FAQ.md